### ISSUE TITLE
Do not lint file extension on folder

### ISSUE THREAD

**Issue by Soullivaneuh** [Pryz/yaml-lint#9]
The folder argument is nearly unusable because the folder can contains other file than yaml files.

See: https://travis-ci.org/sonata-project/SonataTranslationBundle/jobs/135808857#L216

It should be ignored. Or at least, have an option to ignore this check like `--ignore-extension`.

**Comment by Pryz** [comment]
Hi @Soullivaneuh,

Thanks for your comment and sorry for not giving you any answer sooner.

I've just pushed a new version (0.0.9) with an option (-i) to ignore files which are not yaml files. I've tested it against SonataTranslationBundle and it seems to do the job.

Sample :

`File : ./Resources/translations/SonataTranslationBundle.en.xliff, Ignored
File : ./Resources/translations/SonataTranslationBundle.it.xliff, Ignored
File : ./Resources/translations/SonataTranslationBundle.hu.xliff, Ignored
File : ./Resources/translations/SonataTranslationBundle.de.xliff, Ignored
File : ./Resources/translations/SonataTranslationBundle.es.xliff, Ignored
File : ./Resources/config/config.yml, Syntax OK
File : ./Resources/config/service_phpcr.xml, Ignored
File : ./Resources/config/service.xml, Ignored
File : ./Resources/config/twig.xml, Ignored
`
